imageregistry/v1/types.go

```diff
@@ -306,6 +306,78 @@ type ImageRegistryConfigStorageIBMCOS struct {
 	ServiceInstanceCRN string `json:"serviceInstanceCRN,omitempty"`
 }
 
+// EndpointAccessibility defines the Alibaba VPC endpoint for storage
+type EndpointAccessibility string
+
+// AlibabaEncryptionMode defines an enumerable type for the encryption mode
+type AlibabaEncryptionMode string
+
+const (
+	// InternalEndpoint sets the VPC endpoint to internal
+	InternalEndpoint EndpointAccessibility = "Internal"
+	// PublicEndpoint sets the VPC endpoint to public
+	PublicEndpoint EndpointAccessibility = "Public"
+
+	// ClearText is an AlibabaEncryptionMode. This means no encryption
+	ClearText AlibabaEncryptionMode = "ClearText"
```

Comment:
is ClearText a term in Alibaba? Elsewhere afaik we use "None". Also "PlainText" is more a standard term.

Reply:
I don't think ClearText is a verb. I can use `None` or `PlainText`. Will use `PlainText`